waypoints:
  waypoint0:
    x: -2.1241638023639098e-05
    y: -1.4001326235302258e-05
    theta: 0.00023190090351881331
  waypoint1:
    x: 0.3243463349002115
    y: 0.2184188194671135
    theta: -0.5335178136985893
  waypoint2:
    x: -0.30492345532115894
    y: 0.5498244486547555
    theta: -0.4993549205756817
  waypoint3:
    x: -0.7298361168906589
    y: -0.378806634781206
    theta: -0.43277065657574804
  waypoint4:
    x: 0.1106579420983111
    y: -0.7228116059260217
    theta: -0.3085628374241598
  waypoint5:
    x: 0.08202233206825535
    y: 0.5099619593724343
    theta: 0.016810405370490264
